<a href="https://colab.research.google.com/github/suhasshettigark/AIML_Project/blob/main/Heart_Disease_Day2.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [ ]:
import pandas as pd
df = pd.read_csv('https://talentcocomedia.s3.amazonaws.com/ml-assets/heart-disease.csv')
df.head()

age  sex  cp  trestbps  chol  fbs  restecg  thalach  exang  oldpeak  slope  \
0   63    1   3       145   233    1        0      150      0      2.3      0   
1   37    1   2       130   250    0        1      187      0      3.5      0   
2   41    0   1       130   204    0        0      172      0      1.4      2   
3   56    1   1       120   236    0        1      178      0      0.8      2   
4   57    0   0       120   354    0        1      163      1      0.6      2   

   ca  thal  target  
0   0     1       1  
1   0     2       1  
2   0     2       1  
3   0     2       1  
4   0     2       1

In [ ]:
df.shape #rows , columns

(303, 14)

#Features

1.age = age in years \
2.sex = sex (1 = male; 0 = female) \
3.cp = chest pain type \
-- Value 1: typical angina \
-- Value 2: atypical angina \
-- Value 3: non-anginal pain \
-- Value 4: asymptomatic \
4.trestbps = resting blood pressure (in mm Hg on admission to the hospital) \
5.chol = serum cholestoral in mg/dl \
6.fbs = (fasting blood sugar > 120 mg/dl) (1 = true; 0 = false) \
7.restecg = resting electrocardiographic results \
-- Value 0: normal \
-- Value 1: having ST-T wave abnormality (T wave inversions and/or ST elevation or depression of > 0.05 mV) \
-- Value 2: showing probable or definite left ventricular hypertrophy by Estes' criteria \
8.thalach = maximum heart rate achieved \
9.exang: exercise induced angina (1 = yes; 0 = no) \
10.oldpeak = ST depression induced by exercise relative to rest \
11.slope = the slope of the peak exercise ST segment \
-- Value 1: upsloping \
-- Value 2: flat \
-- Value 3: downsloping \
12.ca = number of major vessels (0-3) colored by flourosopy \
13.thal: 3 = normal; 6 = fixed defect; 7 = reversable defect \
14.the predicted attribute
